src/components/snackbar/key-action-snackbar/styles/SKeyActionSnackbarKey.tsx

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from "styled-components";
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../../constants/styles/mediaquerys";
+
+interface ISKeyActionSnackbarKey {}
+
+const Bronze = css<ISKeyActionSnackbarKey>`
+  height: 16px;
+  padding: 0 1.5px;
+  background-color: #f7f9fa;
+  font-size: 11px;
```

Comment:
@nicolasshiken  is should be a predefined font var to use

Reply:
I found an issue with how the fonts are created and created a ticket [here](https://linear.app/caisy/issue/CY-396/bug-fonts-have-incorrect-formatnaming-on-design-tokens), I'll finish this ticket first so I can use font vars wherever are needed!

Reply:
Just updated this with the new CSSVariables